test/template.js

```diff
@@ -15,37 +15,37 @@ describe('TemplateGenerator', function () {
       it('return correct zip filename for https link', function () {
         let result = templateGenerator.getExternalProject("https://github.com/embark-framework/embark");
         assert.strictEqual(result.url, "https://github.com/embark-framework/embark/archive/master.zip");
-        assert.strictEqual(result.filePath, ".embark/templates/embark-framework/embark/archive.zip");
+        assert.strictEqual(result.filePath.replace(/\\/g,'/'), ".embark/templates/embark-framework/embark/archive.zip");
```

Comment:
Doesn’t the stdlib have something that deals with this?

Reply:
No that I know. searched a bit now but didn't find anything